crate_universe/test_data/serialized_configs/BUILD.bazel

```diff
@@ -32,6 +32,7 @@ write_file(
                 "x86_64-pc-windows-msvc",
                 "x86_64-apple-darwin",
             ],
+            transition_opt = True,
```

Comment:
Can you write an integration test for this behaviour?

This should be observable; from a `cargo_build_script` you can get `std::env::var("OPT_LEVEL")` which will be `0` for fastbuild and `3` for opt.

Hopefully we can in two different crate_universe invocations, one which transitions and one which doesn't, verify that the opt levels are set correctly.